Components/model.css

```diff
@@ -0,0 +1,24 @@
+.model-container {
+  position: relative;
+}
+
+.model-pop-up {
+    border : solid  var(--blue) 1px;
+    margin : 1rem;
+    z-index : 9;
+    padding : 0.5rem 1rem;
```

Comment:
You can use multiples of 4 or 8 to to give padding and lengths value. It is a standard practice in industry like instead of 0.5rem you can give 0.4 rem

Reply:
avoid extra spaces